### PR TITLE
Bengali Language added. Issue #969

### PR DESCRIPTION
**Pull request by arnav-mandal1234**
Issue #969
Bengali Language added.

### CODE REVIEW

File: docs/INSTALLATION_LOCAL_BENGALI.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+### 3. ইনস্টলেশনের নির্ভরতা
+
+`npm install`
+
+![npm install](./img/npmistall.png)
+
+আউটপুট: * (নমুনা) *
+
+
+    npm WARN deprecated tar.gz@0.1.1: â ï¸  WARNING â ï¸ tar.gz module has been deprecated and your application is vulnerable. Please use tar module instead: https://npmjs.com/tar
```
Comment (rishiraj824) [inline docs/INSTALLATION_LOCAL_BENGALI.md]: @arnav-mandal1234 Can you please explain why are these required?

Comment (arnav-mandal1234) [reply docs/INSTALLATION_LOCAL_BENGALI.md]: @rishiraj824 There was an issue with adding more language and I have translated the English Docs to Bengali.

Comment (rishiraj824) [reply docs/INSTALLATION_LOCAL_BENGALI.md]:
```
npm WARN deprecated tar.gz@0.1.1: â ï¸  WARNING â ï¸ tar.gz module has been deprecated and your application is vulnerable. Please use tar module instead: https://npmjs.com/tar
```
Ok, but this text comes in the between the code. I think we can remove this.

Comment (arnav-mandal1234) [reply docs/INSTALLATION_LOCAL_BENGALI.md]: Its the part of the code Please follow the English instruction carefully.

**Comment by YashKumarVerma** [comment]
Branch out of date
